wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -130,7 +134,9 @@ public final class Constants {
                 FEE_NETWORK_SUFFIX = "-testnet"; // use the same fee file as testnet
                 WALLET_NAME_CURRENCY_CODE = "tdash";
                 org.dash.wallet.common.util.Constants.EXPLORE_GC_FILE_PATH = "explore/explore-devnet.db";
-                SUPPORTS_PLATFORM = true;
+                SUPPORTS_PLATFORM = !is32Bit;
+                SUPPORTS_INVITES = false;
+                SUPPORTS_TXMETADATA = false;
```

Comment:
We haven't made devnet builds in a long time, but let's update the feature flags.